pkgs/by-name/co/coal/package.nix

```
@@ -15,15 +16,14 @@
 }:
 
 stdenv.mkDerivation (finalAttrs: {
-  pname = "hpp-fcl";
-  version = "2.4.5";
+  pname = "coal";
+  version = "3.0.0";
 
   src = fetchFromGitHub {
-    owner = "humanoid-path-planner";
-    repo = "hpp-fcl";
-    rev = "v${finalAttrs.version}";
-    fetchSubmodules = true;
-    hash = "sha256-0OORdtT7vMpvK3BPJvtvuLcz0+bfu1+nVvzs3y+LyQw=";
+    owner = "coal-library";
+    repo = "coal";
+    rev = "refs/tags/v${finalAttrs.version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
You can use the new `tag` argument now. Not a blocker though.